config/sync/user.role.site_editor.yml

```diff
@@ -0,0 +1,53 @@
+uuid: b35dd6b6-930d-4b4c-aeb2-443a0ba6fb59
+langcode: en
+status: true
+dependencies: {  }
+id: site_editor
+label: 'Site Editor'
+weight: -2
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access environment indicator'
+  - 'access file_browser entity browser pages'
+  - 'access files overview'
+  - 'access image_browser entity browser pages'
+  - 'access media overview'
+  - 'access media_browser entity browser pages'
+  - 'access patterns page'
+  - 'access video_browser entity browser pages'
```

Comment:
the entity browser pages are going to be removed after stanford_media update is in place

Reply:
TY.